File: 05-engenharia-software/modelagem-sistemas/uml-poo/Codigos/Aula 4/conta/Exemplo banco/c-sharp/poo_conta_01/Servicos/Conta.cs
namespace poo_conta
{
    public class Conta
    {
        public Conta(int agencia, int numeroConta, int Saldo)
        {
            this.Agencia = agencia;
            this.NumeroConta = numeroConta;
            this.Saldo = Saldo;
        }
        
        public int Agencia { get; set; }
        public int NumeroConta { get; set; }
        private float _saldo;
        public float Saldo { 
            get{ return _saldo; }
            set{
               _saldo = value; 
            } 
        }
        
        public void Depositar(float dinheiro)
        {
            if (dinheiro > 0)
            {
                Saldo = dinheiro;
            }
        }
    }
}

File: 02-programacao/php/02-poo/03/banco/poo/Conta.php
<?php
    class Conta {

        public int $Agencia;
        public int $NumeroConta;
        private float $_saldo;

        private function setSaldo(float $saldo){
            $this->_saldo = $saldo;
        }

        function getSaldo() {
            return $this->_saldo;
        }

        function Depositar(float $dinheiro)
        {
            if ($dinheiro > 0) {
                $this->setSaldo($dinheiro); 
            }
        }

    }